### PR TITLE
chore: fix AIO SSL; bump admin-mgr

### PR DESCRIPTION
**Pull request by pcnc**
* `configure-pgbouncer.sh` runs after `configure-adminapi.sh`, overwriting its settings
* admin-mgr fixes for LSN checkpointing

### CODE REVIEW

**Review by Crispy1975 — APPROVED** [review]

File: docker/all-in-one/init/configure-pgbouncer.sh [hunk 1]
```diff
@@ -37,10 +37,12 @@ if [ "${DATA_VOLUME_MOUNTPOINT}" ]; then
   chown -R pgbouncer:pgbouncer "/etc/pgbouncer-custom"
   chown -R pgbouncer:pgbouncer "${PGBOUNCER_CUSTOM_DIR}"
   chmod -R g+rx "${PGBOUNCER_CUSTOM_DIR}"
+  chmod -R 0775 "${PGBOUNCER_DIR}"
```
Comment (thebengeu) [inline docker/all-in-one/init/configure-pgbouncer.sh]: Should this be `PGBOUNCER_CUSTOM_DIR`? Do we need to `chmod` twice?

File: docker/all-in-one/init/configure-pgbouncer.sh [hunk 2]
```diff
@@ -37,10 +37,12 @@ if [ "${DATA_VOLUME_MOUNTPOINT}" ]; then
...
 
   rm -rf "/etc/pgbouncer"
   ln -s "${PGBOUNCER_DIR}" "/etc/pgbouncer"
   chown -R pgbouncer:pgbouncer "/etc/pgbouncer"
   chown -R pgbouncer:pgbouncer "${PGBOUNCER_DIR}"
   chmod -R g+wrx "${PGBOUNCER_DIR}"
+  chmod -R 0775 "${PGBOUNCER_DIR}"
```
Comment (thebengeu) [inline docker/all-in-one/init/configure-pgbouncer.sh]: Do we need to `chmod` twice?

**Review by thebengeu — APPROVED** [review]

File: docker/all-in-one/init/configure-pgbouncer.sh (hunk 1 above)
Comment (pcnc) [reply docker/all-in-one/init/configure-pgbouncer.sh]: Yep, image just built and tested it - this is indeed a typo and should be `PGBOUNCER_CUSTOM_DIR`

Comment (pcnc) [reply docker/all-in-one/init/configure-pgbouncer.sh]: No need to chmod twice - added `w` perms to the initial chmod

**Pull request merged**